### ISSUE TITLE
Support for git push and git pull?

### ISSUE THREAD

**Issue by NemoAlex** [thisconnect/nodegit-kit#14]
Any possible to add support for push and pull (fetch & merge)?
Is that in your plan?

**Comment by thisconnect** [comment]
Yes, I have remote locally but not yet fetch or merge. If I ever make travis build again .. :/

**Comment by thisconnect** [comment]
@NemoAlex added super simple push and pull and git remote support. This currently only works with the current branch. Before continuing with push and pull proper support for git branch (#2) is needed .

**Comment by NemoAlex** [comment]
Great! 
I'll check it out. Thank you very much. Super!

**Comment by thisconnect** [comment]
`npm i thisconnect/nodegit-kit#dev`

**Comment by thisconnect** [comment]
correction: only pull uses current branch, push is hardcoded to master 

https://github.com/thisconnect/nodegit-kit/commit/b6042cbf076a27bab9d9c062c69171922f9fae3f#diff-539df6289b63785f3c1f4660c1db4107R27
